dev-python/google-auth-oauthlib/google-auth-oauthlib-0.2.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+inherit distutils-r1
+
+DESCRIPTION="Google Authentication Library"
+HOMEPAGE="https://pypi.org/project/google-auth-oauthlib/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-python/google-auth[${PYTHON_USEDEP}]
+	>=dev-python/requests-oauthlib-0.7.0[${PYTHON_USEDEP}]
+"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+"
```

Comment:
I see upstream has python tests enabled, could you implement those in the ebuild?

Reply:
Sure, I'll pick a python ebuild with the `test` flag to learn how to do this and I'll add it.